proto/elys/tokenomics/airdrop.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package elysnetwork.elys.tokenomics;
+
+option go_package = "github.com/elys-network/elys/x/tokenomics/types";
+
+message Airdrop {
+  string intent = 1; 
+  uint64 amount = 2; 
+  string authority = 3;
```

Comment:
@cosmic-vagabond It would be useful to have descriptions of what all the new data structure fields are. Either in a spec or the readme or in comments somewhere. Just to help with readability so we don't have to search around to see what does what.

Reply:
@AustinoBombino makes sense I will add some specs document to the code.